[windows] Compiler doesn't build if the username has a space

When the script tries to compile build tools, the bin log argument is without quotes

**Expected behavior**

Run `Build.cmd` without errors

**Actual behavior**

```powershell
MSBUILD : error MSB1008: Only one project can be specified.
```

--- a/eng/build-utils.ps1
+++ b/eng/build-utils.ps1
@@ -247,7 +247,7 @@ function Make-BootstrapBuild() {
     $args = "build $buildToolsProject -c $bootstrapConfiguration -v $verbosity" + $argNoRestore + $argNoIncremental
     if ($binaryLog) {
         $logFilePath = Join-Path $LogDir "toolsBootstrapLog.binlog"
-        $args += " /bl:$logFilePath"
+        $args += " /bl:`"$logFilePath`""
     }
     Exec-Console $dotnetExe $args
 
@@ -260,7 +260,7 @@ function Make-BootstrapBuild() {
     $args = "build $protoProject -c $bootstrapConfiguration -v $verbosity -f $bootstrapTfm" + $argNoRestore + $argNoIncremental
     if ($binaryLog) {
         $logFilePath = Join-Path $LogDir "protoBootstrapLog.binlog"
-        $args += " /bl:$logFilePath"
+        $args += " /bl:`"$logFilePath`""
     }
     Exec-Console $dotnetExe $args
 

